modules/docker/inspect_test.go

```diff
@@ -0,0 +1,126 @@
+package docker
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/shell"
+	"github.com/stretchr/testify/require"
+)
+
+const dockerInspectTestImage = "nginx:1.17-alpine"
+
+func TestInspect(t *testing.T) {
+	t.Parallel()
+
+	// append timestamp to container name to allow running tests in parallel
+	name := "inspect-test-" + random.UniqueId()
+
+	// running the container detached to allow inspection while it is running
+	options := &RunOptions{
+		Detach: true,
+		Name:   name,
+	}
+
+	// remove the local image if it exists to force Docker to pull it
```

Comment:
Why do we want to force a `pull`?

Reply:
I didn't catch this error when developing because I already had the image downloaded and cached. 

I added this test to validate that scenario, in case something changes (either in Terratest or in Docker) so that we can have the same behavior in CI and local testing, catching it quicker. 

Do you think it would be better to document that in the comment (line 27) too?

Reply:
Well, the test _already_ failed due to this issue, so our testing was good enough to catch it. Sure, we didn't catch it until CI, but IMO, that's a better trade-off than (a) deleting a local Docker image, which would be an unexpected thing for a test to do and (b) making each test run slower.

Reply:
That makes sense @brikis98! I've removed the `rmi` command in [73ef44b](https://github.com/gruntwork-io/terratest/pull/492/commits/73ef44be58fe786ebfcb655297d7c06391712ad7).